### PR TITLE
KEP-1420 Added and corrected missing build steps for Ubuntu

### PR DESCRIPTION
**Pull request by nickroeker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,18 +41,31 @@ $ brew install ccache
...
+
+```text
+$ sudo apt-get install curl libcurl4-openssl-dev
+$ sudo apt-get install cmake
+```
+If your system packages don't have a new enough version, you can install a
+different CMake into `~/mycmake/` to avoid overwriting your system's `cmake`.
 
 ```text
+$ sudo apt-get install curl libcurl4-openssl-dev
```
Comment (ebruck) [inline README.md]: curl comes automatically with the dev package?

Comment (nickroeker) [reply README.md]: It does not unfortunately, assuming you mean `libcurl4-openssl-dev`. At least not in 18.04's default packages? I had to separately install it in a fresh VM after doing all other setup.

Comment (ebruck) [reply README.md]: strange (pulled from 19.04):

Package: libcurl4-openssl-dev
Source: curl
Version: 7.64.0-2ubuntu1
Architecture: amd64
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Installed-Size: 1472
**Depends: libcurl4 (= 7.64.0-2ubuntu1)**

Comment (nickroeker) [reply README.md]: You're correct of course, I must've goofed it somehow in the first round. `curl` itself seems to only be required by the instructions as well, as the pulled-in libcurl does just fine. Will update.

**Pull request merged**